common/src/View/ViewPreferencePane.cpp

```diff
@@ -180,6 +189,10 @@ namespace TrenchBroom {
             layout->addRow("Edges", m_edgeColorButton);
 
             layout->addSection("Texture Browser");
+            layout->addRow("Text", m_browserTextColorButton);
```

Comment:
Since these colors apply to all browsers and not just the texture browser, could you change the section title to read just "Browser"?

Reply:
Actually, I've decided to go a step further and make all color preferences editable:

![trenchbroom-color-prefs](https://user-images.githubusercontent.com/7252064/99951080-8ed71b80-2d8e-11eb-82c4-bab21f82d9c9.jpg)

This still requires some additional work. I'll make a separate pull request once it's done.

Reply:
I'm not sure I agree with making all of these editable. I'd like to keep the preferences simple in general. Next thing people will ask for is theming, and then we'll be dealing with two kinds of themes (UI and this).

Reply:
This makes me wonder why all those colors were created as `Preference<Color>` with valid descriptions, not just `Color`...

Reply:
To have the option to make them customizable in the future and to have only one way of accessing such colors (instead of hard coding them).

Reply:
@ericwa What do you think? On one hand, some users will surely be happy about this, on the other hand, MUH UX PRINCIPLES.